app/integration_action.go

```diff
@@ -573,6 +573,10 @@ func (a *App) OpenInteractiveDialog(request model.OpenDialogRequest) *model.AppE
 		return err
 	}
 
+	if dialogErr := request.IsValid(); dialogErr != nil {
+		return dialogErr
+	}
+
 	request.TriggerId = clientTriggerId
```

Comment:
@hanzei adding the isValid method in openDialog is leading to error, as the dialogErr returns an error and the openInteractiveDialog returns model.AppError. Any suggestions??

Reply:
Using `model.NewAppError` you can wrap the returned `error` into an `model.AppError` similar how it's done in https://github.com/mattermost/mattermost-server/pull/16358/files/796c8226667c30d0d1becae347a7297f87d78ca2#diff-47384b35a1503accad7764e31e11043e2a8aaa220a7fa8b03009f2704346512cR597-R599. That also means that you have to add a new translation string to ``i18n/en.json`.

Reply:
this points to the current PR. Can you please direct me to the correct url

Reply:
The line I wanted to point out is: https://github.com/mattermost/mattermost-server/blob/796c8226667c30d0d1becae347a7297f87d78ca2/app/integration_action.go#L596-L599